.travis.yml

```diff
@@ -86,14 +86,14 @@ script:
     export PATH="${PATH}:$(pwd)/artifacts"
     if [ `uname` = "Darwin" ]
     then
-      stack build --test --no-run-tests --copy-bins --local-bin-path ./artifacts;
+      stack build --test --no-run-tests --copy-bins --local-bin-path ./artifacts --haddock --no-haddock-deps;
```

Comment:
Here (and a few other places), the `--haddock` flag has been added.

This builds the haddocks when building spago.  The build will fail if the haddocks for `spago` cannot be generated.

By default, all the haddocks for all depedencies are also built.  Unfortunately, the haddocks for [github](http://hackage.haskell.org/package/github) don't build correctly, so I had to use the `--no-haddock-deps` flag.  This makes sure haddocks all all dependencies are not built.

Reply:
The bug in the haddocks for github appears to be fixed in the `master` branch on the repo, so `--no-haddock-deps` could potentially be removed here in the future.

Reply:
Yeah, building Haddocks should work again once we are back on the latest GHC/cabal/haddock versions (currently pinning to LTS-12 due to supporting older macOS versions), but having `--no-haddock-deps` should be fine for now!